docs/framework/tools/al-exe-assembly-linker.md

```diff
@@ -48,7 +48,7 @@ al sources options
+|**/link**[`resource`]:`file`[,`name`[,`target`[,`private`]]]|Links a resource file to an assembly. The resource specified by `file` becomes part of the assembly; the file is not copied. The `file` parameter can be in any file format. For example, you can specify a native DLL as the `file` parameter. This will make the native DLL part of the assembly so that it can be installed into the global assembly cache and accessed from managed code in the assembly. You can also do this by using the **/linkresource** compiler option. For more information, see [/linkresource (C# Compiler Options)](~/docs/csharp/language-reference/compiler-options/linkresource-compiler-option.md).<br /><br /> The `name` parameter is an internal identifier for the resource. The `target` parameter specifies a path and file name into which Al.exe copies the `file`*.* After copying, Al.exe compiles `target` into an assembly. By default, resources are public in the assembly (visible to other assemblies). Specifying `private` makes the resource not visible to other assemblies.<br /><br /> If `file` is a .NET Framework resource file created, for example, by the Resource File Generator (Resgen.exe) or in the development environment, it can be accessed with members in the <xref:System.Resources> namespace. For more information, see <xref:System.Resources.ResourceManager>. For all other resources, use the `GetManifestResource` * methods in the <xref:System.Reflection.Assembly> class to access the resource at run time.<br /><br /> If only resource files are passed to Al.exe, the output file is a satellite resource assembly.|  
```

Comment:
@svick I see a few little things on this topic not directly relevant to your PR. I'll get 'um ... I'm looking for some small fill-in thingies to do before I go. I don't want to hold this set of changes up. I'll work it on:  https://github.com/dotnet/docs/issues/2739